### PR TITLE
Second Commit by Daniel

### PR DESCRIPTION
**Pull request by daniel-lyn**
I changed the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 Python script for enabling web scraping.
 Modified Date: 14/1/2022
 George Brown College
+Machine Learning Lab Exercise - Github
```
Comment (henrywan1920) [inline README.md]: It should be "You are correct.."

**Review by henrywan1920 — COMMENTED** [review]
Code review first.

File: README.md (hunk 1 above)
Comment (daniel-lyn) [reply README.md]: ok will change now

**Comment by daniel-lyn** [comment]
You are correct

**Pull request merged**